src/diffpy/srmise/baselines/arbitrary.py

```diff
@@ -43,15 +43,18 @@ def __init__(self, npars, valuef, jacobianf=None, estimatef=None, Cache=None):
         """Initialize an arbitrary baseline.
 
         Parameters
-        npars: Number of parameters which define the function
-        valuef: Function which calculates the value of the baseline
-               at x.
-        jacobianf: (None) Function which calculates the Jacobian of the
+        ----------
+        npars : int
+            Number of parameters which define the function
```

Comment:
start all these with "The", i..e., "The number of parameters...."

This syntax is almost universal and helps to make docstrings easier to read in general.